tests/telemetry/events/dhcp_relay_events.py

```diff
@@ -0,0 +1,88 @@
+#! /usr/bin/env python3
+
+import logging
+import time
+from run_events_test import run_test
+from tests.common.utilities import wait_until
+
+logger = logging.getLogger(__name__)
+tag = "sonic-events-dhcp-relay"
+
+interface = ""
+ipv4_address = ""
+ipv6_address = ""
+
+
+def test_event(duthost, gnxi_path, ptfhost, data_dir, validate_yang):
+    logger.info("Beginning to test dhcp-relay events")
+    try:
+        backup_ip_address(duthost)
+    except Exception:
+        logger.info("Skipping test, no Vlan interface")
+        return
```

Comment:
The thought process is that if we are not able to retrieve information needed for this testcase such as the ipv4/v6 address and the corresponding Vlan interface, we should not fail this testcase as there is no fault with the actual test. I would like to skip this testcase in that situation but I understand that it currently looks like a success vs a skip.

Reply:
Exception is too general type, and some bug inside test script will lead to a Exception and blindly pass the testcase.

Since you anticipated "not able to retrieve information needed for this testcase such as the ipv4/v6 address" may be possible, could you catch this situation by more specific exception type, or even without exception?